client/src/Game.cpp

```diff
@@ -84,31 +68,36 @@ void Game::processEvents() {
 }
 
 void Game::update(sf::Time deltaTime) {
-    player.update(deltaTime);
-
-    playerCommands.updatePlayer(deltaTime);
+    if (!player.expired()) {
```

Comment:
If it is expired, that's mean that something really bad happened. Just let it crash.